rootSeed.js

```diff
@@ -0,0 +1,21 @@
+const { Seeder } = require('mongo-seeding');
+
+
+const mongoose = require('mongoose');
+mongoose.connect("mongodb://localhost:27017/LMS" , {useNewUrlParser: true, useUnifiedTopology: true})    
+const db = mongoose.connection
+db.on('open',()=>{
+    console.log('connected with the database');
+})
+
+let {Users, Roles} = require('./models/users')
+
```

Comment:
first check if the role already exists in the Roles collection or not. If it doesn't exist create the role else do nothing